backstory/pom.xml

```diff
@@ -68,6 +68,13 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
```

Comment:
The mockito-inline package should be used for consistency with the one being used already in StoryManager as discussed earlier.